DigitalLearningSolutions.Web/ViewComponents/FileInputViewComponent.cs

```diff
@@ -1,28 +1,34 @@
-﻿namespace DigitalLearningSolutions.Web.ViewComponents
```

Comment:
Line endings are correct in this file, I've fixed the mixed line endings to be CRLF.